### PR TITLE
Adds a dispatch data source that replaces its current value with a ne…

### PR DESCRIPTION
**Pull request by ktopley-apple**
…w value.

(Radar 29289890)

### CODE REVIEW

**Comment by ktopley-apple** [comment]
@mwwa please review

**Review by mwwa — APPROVED** [review]

File: stdlib/public/SDK/Dispatch/Source.swift [hunk 1]
```diff
@@ -299,35 +303,46 @@ public extension DispatchSourceFileSystemObject {
 }
 
 public extension DispatchSourceUserDataAdd {
-	/// @function mergeData
+	/// @function add
```
Comment (jrose-apple) [inline stdlib/public/SDK/Dispatch/Source.swift]: Ah, if you're fixing these up, you should probably use Swift doc comment syntax. Swift does not recognize HeaderDoc.

**Comment by ktopley-apple** [comment]
HeaderDoc comments replaced with Swift comments, as suggested

File: stdlib/public/SDK/Dispatch/Source.swift (hunk 1 above)
Comment (ktopley-apple) [inline stdlib/public/SDK/Dispatch/Source.swift]: HeaderDoc comments replaced with Swift comments, as suggested

**Comment by ktopley-apple** [comment]
@swift-ci please smoke test

**Comment by shahmishal** [comment]
@swift-ci please smoke test

**Pull request closed without merging**